pallets/rollup_main_contract/src/mock.rs

```diff
@@ -0,0 +1,78 @@
+use crate as main_contract_pallet;
```

Comment:
we don't deploy the contract so directly name and trait name should be updated.
`pallets/zkrollup_verifier` is good I think

Reply:
It's not verifier for sure, but I'll think about better naming!